markdowntext/src/main/java/dev/jeziellago/compose/markdowntext/MarkdownText.kt

```diff
@@ -19,22 +32,60 @@ import io.noties.markwon.linkify.LinkifyPlugin
 @Composable
 fun MarkdownText(
     markdown: String,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
+    color: Color = Color.Unspecified,
+    fontSize: TextUnit = TextUnit.Unspecified,
+    letterSpacing: TextUnit = TextUnit.Unspecified,
+    textAlign: TextAlign? = null,
+    maxLines: Int = Int.MAX_VALUE,
+    @FontRes fontResource: Int? = null,
+    style: TextStyle = LocalTextStyle.current
 ) {
-    val textColor = if (MaterialTheme.colors.isLight) {
-        android.graphics.Color.BLACK
-    } else {
-        android.graphics.Color.WHITE
+    val textColor = color.takeOrElse {
+        style.color.takeOrElse {
+            LocalContentColor.current.copy(alpha = LocalContentAlpha.current)
+        }
     }
+    val mergedStyle = style.merge(
+        TextStyle(
+            color = textColor,
+            fontSize = fontSize,
+            textAlign = textAlign,
+            letterSpacing = letterSpacing
+        )
+    )
+
     val context = LocalContext.current
     val markdownRender = remember { createMarkdownRender(context) }
     val markdownText = TextView(LocalContext.current).apply {
-        setTextColor(textColor)
+        setTextColor(textColor.toArgb())
+        setMaxLines(maxLines)
+        setTextSize(TypedValue.COMPLEX_UNIT_DIP, mergedStyle.fontSize.value)
+        if(mergedStyle.letterSpacing.type != TextUnitType.Unspecified) setLetterSpacing(mergedStyle.letterSpacing.value)
+        when (textAlign) {
+            TextAlign.Left -> View.TEXT_ALIGNMENT_TEXT_START
+            TextAlign.Right -> View.TEXT_ALIGNMENT_TEXT_END
+            TextAlign.Center -> View.TEXT_ALIGNMENT_CENTER
+            TextAlign.Justify -> null
+            TextAlign.Start -> View.TEXT_ALIGNMENT_TEXT_START
+            TextAlign.End -> View.TEXT_ALIGNMENT_TEXT_END
+            null -> null
+        }?.let {
+            textAlignment = it
+        }
```

Comment:
Should be simplified:
```suggestion
        when (textAlign) {
            TextAlign.Left, TextAlign.Start -> View.TEXT_ALIGNMENT_TEXT_START
            TextAlign.Right, TextAlign.End -> View.TEXT_ALIGNMENT_TEXT_END
            TextAlign.Center -> View.TEXT_ALIGNMENT_CENTER
            else -> null
        }?.let { align ->
            textAlignment = align
        }
```